### ISSUE TITLE
The GMSACredentialSpec CRD is already included in the deploy-gmsa-webhook script

### ISSUE THREAD

**Issue by idofl** [kubernetes/website#26934]
The script linked in the [Install webhooks to validate GMSA users](https://kubernetes.io/docs/tasks/configure-pod-container/configure-gmsa/#install-webhooks-to-validate-gmsa-users) section includes the step to deploy the gmsa-crd.yml file that is referenced in the [Install the GMSACredentialSpec CRD](https://kubernetes.io/docs/tasks/configure-pod-container/configure-gmsa/#install-the-gmsacredentialspec-crd) section.

Please consider joining those two sections into one, and remove the direct link to the gmsa-crd.yaml file.

The instruction to deploy the gmsa crd is here:
https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/deploy-gmsa-webhook.sh#L140

**Comment by sftim** [comment]
/sig windows
@kubernetes/sig-windows-misc - what's the right approach here?

**Comment by marosset** [comment]
/assign @ddebroy @jsturtevant 
Can one of you take a look here?

**Comment by ddebroy** [comment]
Seems like we need an extra new-line before 
```
A script can be used to deploy and configure the GMSA webhooks and associated objects ...
```
to clarify the fact that the script handles the steps mentioned in both of the above sections - [1] Install the CRD and [2] Install the Webhook - rather than the latter section only. @idofl do you think that will clarify things?

**Comment by idofl** [comment]
The first section has the following sentence:
"Download the GMSA CRD YAML and save it as gmsa-crd.yaml. Next, install the CRD with kubectl apply -f gmsa-crd.yaml"

Since the script handles this, there is no need to add this sentence.

A new line will help to separate the sentence about running the script from the two previous sections, but it only mentions the script, without requesting to download it and run it. I would rephrase the sentence to say:
"Download the deploy-gmsa-webhook.sh script. The script creates the webhooks and associated objects mentioned above in your cluster. The script can be run with a --dry-run=server option to allow you to review the changes that would be made to your cluster."
